server/routes/game.js

```diff
@@ -1,5 +1,21 @@
 const router = require('express').Router();
 const {GameTable, PlayerSession} = require('../db');
+const keys = require('../../apikey')
+//getting keys from heroku config
+const agoraKeys = keys || {
```

Comment:
since we don't have keys for enough rooms, and we're going to do testing in probably more than 3 games, should we just hard code this to always use key 1 for now? I figure it's ok for demonstration purpose to have every table in the same room, since only one table will ever be in use.